lib/resource/Tenant.js

```diff
@@ -220,4 +220,21 @@ Tenant.prototype.getCustomData = function getCustomData(/* [options,] callback *
   return this.dataStore.getResource(this.customData.href, args.options, require('./CustomData'), args.callback);
 };
 
+/**
+ * Retrieves all the {@link IdSiteModel} resources for this resource.
+ *
+ * @param {CollectionQueryOptions} [collectionQueryOptions]
+ * Options for querying, paginating, and expanding the collection.
+ *
+ * @param {Function} callback
+ * The function to call when then the operation is complete. Will be called
+ * with the parameters (err, {@link CollectionResource}). The collection will
+ * be a list of {@link IdSiteModel} objects.
+ *
+ */
+Tenant.prototype.getIdSiteModels = function getTenantIdSiteModels(/* [options,] callback */) {
```

Comment:
I think this should be `getIdSites()` instead of `getIdSiteModels()`. Your reasoning is correct. But the underlying model is named incorrectly so we'll have to fix that in another PR.

https://github.com/stormpath/stormpath-sdk-node/issues/537